### DISCUSSION TITLE
Add polygroups from materials feature please!

### DISCUSSION THREAD

**Discussion by 130N1D** [JoseConseco/GoB#312] [Ideas]
Add polygroups from materials feature please!

**Comment by kromar** [comment]
Hi @130N1D i added the export from blender to zbrush in this branch, (v3_5_80) only the export from blender to zbrush works for now but i will add the polygroups to materials from zbrush to blender later.
Let me know if you encounter any issues.
https://github.com/kromar/blender_GoB/tree/material-to-polygroups
https://github.com/kromar/blender_GoB/archive/refs/heads/material-to-polygroups.zip

**Comment by 130N1D** [comment]
Wow, that was fast.. but unfortunately I can't export a model with materials to ZBrush 2022, the app freeze when I export any model 
https://imgur.com/U0G4IU1
READING GOZ FILE... then freeze

**Comment by kromar** [comment]
Thanks for reporting back, the crash should be fixed in v3_5_82, the same link should be valid for that.
